<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="tm5" ignored="0" name="Test" tests="1" failures="1" timestamp="22 Jun 2018 05:42:18 GMT" time="51.878" errors="0">
  <testcase name="Tc_01_ValidateLoginCredentials" time="48.494" classname="com.TestCases.Tc_01_ValidateLogin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat com.codoid.products.fillo.SelectExcelUtil.runSelectQuery(SelectExcelUtil.java:46)at com.codoid.products.fillo.Select.getRecordset(Select.java:61)at com.codoid.products.fillo.Connection.executeQuery(Connection.java:56)at com.GenericFunctions.GenericFunctions.senddata(GenericFunctions.java:368)at com.TestCases.Tc_01_ValidateLogin.Tc_01_ValidateLoginCredentials(Tc_01_ValidateLogin.java:35)... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Tc_01_ValidateLoginCredentials -->
</testsuite> <!-- Test -->
